src/main/java/org/commonjava/event/common/IndyEvent.java

```diff
@@ -0,0 +1,29 @@
+package org.commonjava.event.common;
+
+import org.commonjava.event.common.EventMetadata;
+
+import java.util.UUID;
+
+public class IndyEvent
+{
+
```

Comment:
we might need a versionId. Initially it is empty. Once we want to evolve it, set to sth like "v2, beta-1,...". The idea is we can send different versioned events to same topic. The old services can ignore events he does not know. I don't know if this is really needed. wdyt?

Reply:
Yeah, that should be considered. I will add that later.